Environment-Definitions/ARMTemplates/App_Service_with_Cosmos/environment.yaml

```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=https://github.com/Azure/deployment-environments/releases/download/2022-11-11-preview/manifest.schema.json
+name: App-Service-with-Cosmos
```

Comment:
We might want to change these environment definition names to having a connection to AZD such as 'App-Service-With-Cosmos-AZD' or 'AZD-App-Service-With-Cosmos' so they can be seen when accessing them in AZD CLI or DevPortal. I'll leave this up to Sagar though.

Reply:
Below is my suggestion

1. App-Service-with-Cosmos_AZD-template
2. Container-App-with-Cosmos_AZD-template
3. Function-App-with-Cosmos_AZD-template